armi/operators/operator.py

```diff
@@ -608,12 +608,12 @@ def interactAllEveryNode(self, cycle, tn, excludedInterfaceNames=None):
         excludedInterfaceNames : list, optional
             Names of interface names that will not be interacted with.
         """
-        activeInterfaces = self.getActiveInterfaces("EveryNode")
+        activeInterfaces = self.getActiveInterfaces("EveryNode", excludedInterfaceNames)
         self._interactAll("EveryNode", activeInterfaces, cycle, tn)
 
     def interactAllEOC(self, cycle, excludedInterfaceNames=None):
```

Comment:
If you change this default to be an empty tuple like above, you can get rid of the extra error handling in https://github.com/terrapower/armi/pull/1560/commits/8fe16b2f1a8a8d975b08439d4c19345e660c531b

Reply:
I can fix this, but sadly, some of our internal cases DO explicitly pass `None` in here, so that error-handling will have to remain.

Fixed.